specification/keyvault/resource-manager/Microsoft.KeyVault/preview/2020-04-01-preview/secrets.json

```diff
@@ -141,6 +153,18 @@
             "schema": {
               "$ref": "#/definitions/Secret"
             }
+          },
+          "400": {
```

Comment:
These errors are reused. You made them models above for keyvault.json and should be consistent here as well.